frontend/packages/dev-console/src/components/SearchPage.tsx

```diff
@@ -41,9 +41,7 @@ const Search: React.FC<SearchPageProps> = (props) => {
   const { t } = useTranslation();
   return (
     <>
-      <Helmet>
-        <title data-test="page-title">{t('devconsole~Search')}</title>
```

Comment:
Do we need to retain `data-test="page-title"` here?

Reply:
Doesn't look like it, the only places where I saw `[data-test="page-title"]` being used was for the topology page and for quick starts